config/default/testRunner.conf.php

```diff
@@ -652,6 +652,12 @@
      */
     'item-cache-size' => 3,
 
+    /**
+     * Defines the TimeToLive of items in the cache, in seconds
+     * This is required for caching scenarios
+     * @type integer
+     */
+    'item-store-ttl' => 15 * 60,
```

Comment:
Why must the item validity be so short? What would be the impact of choosing a greater value, e.g. 30 minutes, 2 hours, 8 hours...?

Reply:
I've updated the description.

Reply:
What I really meant was, do you know why the BE uses a short TTL for assets? What is the use case? In production at least, it seems to me that the asset you reference in a compiled test should be invariant, nobody should be changing it.

Reply:
Maybe it's short for security? Since if you know url, you can see asset without any auth cookies. Maybe they are afraid people will share this url with others, or just intercept it. But yes, we should ask BE.
I also still think that current TTL is too small. Why not 1 hour for urls, 30min for itemStore?

Reply:
I think the default value is aligned with the session timeout. And yes, the idea behind the short TTL is to protect the access to the resources. Increasing the value is possible, but it must be aligned with the TTL set for the websource, which is defined on another config file.

Anyway, this is outside of the scope of this ticket. The default value must be lower than the default TTL for the assets. I added recommendation in the documentation related to the pre-caching.